alicloud/resource_alicloud_security_group_test.go

```diff
@@ -98,104 +97,64 @@ func testSweepSecurityGroups(region string) error {
 	return nil
 }
 
-func TestAccAlicloudSecurityGroup_basic(t *testing.T) {
-	var sg ecs.DescribeSecurityGroupAttributeResponse
+func testAccCheckSecurityGroupExists(n string, sg *ecs.DescribeSecurityGroupAttributeResponse) resource.TestCheckFunc {
```

Comment:
I think the method `testAccCheckSecurityGroupExists ` should be removed.

Reply:
this method is referenced by db_instance and fc_service testcase, so hold it temporarily.